### DISCUSSION TITLE
评论编辑功能

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#138] [Ideas]
能不能在前端添加一个编辑/修改功能！

有时候是自己打错字了想修改，或者别人打错了想修改，好像都需要去数据库编辑器，能否在前端控制 修改重新编辑评论呢？？

![QQ截图20220506151634](https://user-images.githubusercontent.com/19650297/167085199-fb7c98fd-5b44-456c-8725-7bb5da6399f3.png)

**Comment by qwqcode** [comment]
已支持评论内容编辑！
